storage/setup.py

```diff
@@ -30,7 +30,7 @@
 release_status = "Development Status :: 5 - Production/Stable"
 dependencies = [
     "google-auth >= 1.9.0, < 2.0dev",
-    "google-cloud-core >= 1.1.0, < 2.0dev",
+    "google-cloud-core >= 1.16.0, < 2.0dev",
```

Comment:
It looks like storage depends on `google-api-core` through `google-cloud-core`. You'll want to make a release of `google-cloud-core` that bumps the `google-api-core` version. https://github.com/googleapis/google-cloud-python/blob/master/core/setup.py

Then you can update this PR with the new `google-cloud-core` version.

Reply:
😛i'm a n00b